rolling-shutter/mocksequencer/processor_test.go

```diff
@@ -0,0 +1,26 @@
+package mocksequencer
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestActivationBlockMap(t *testing.T) {
+	mp := newActivationBlockMap[uint64]()
+	mp.Set(uint64(666), 5)
```

Comment:
explicit casting to `uint64` should be unnecessary here

Reply:
for readability, using `string` values might be a bit easier though (makes it easier to decide between value and block number)

Reply:
Not sure what you mean - make the map a "string" valued map?
I will not implement this for now because I think it's not that important and I don't want to deal with the type-conversion when using that map and comparing against tx-fields (uint64)

Reply:
Sorry, I mean just for this test. Activation block maps are generic, so you could use `string` as the type parameter and store `"first entry"` starting at block 0, `"second entry"` starting at block 5, etc.

Reply:
oh ok that makes sense